fractions/keycloak-microprofile-jwt/src/main/java/org/wildfly/swarm/keycloak/mpjwt/runtime/KeycloakMicroprofileJwtArchivePreparer.java

```diff
@@ -54,6 +63,63 @@ public void process() throws IOException {
         }
     }
 
+    private InputStream getKeycloakJsonFromSystemProperties() {
+        final String thorntailPrefix = "thorntail.keycloak.secure-deployments.[" + archive.getName() + "].";
+        final String swarmPrefix = "swarm.keycloak.secure-deployments.[" + archive.getName() + "].";
+        final String doubleQuote = "\"";
+        final String credentialsSecretProperty = "credentials.secret.value";
+
+        Map<String, String> kcAdapterSystemProps = new LinkedHashMap<>();
+        Properties systemProps = System.getProperties();
+        for (Enumeration<Object> en = systemProps.keys(); en.hasMoreElements();) {
```

Comment:
`Properties` is also a `Map`, so you can iterate its `entrySet()`.

Reply:
Done